### ISSUE TITLE
Feed error when can't resolve remote name

### ISSUE THREAD

**Issue by johnnyramos** [chocolatey/ChocolateyGUI#207]
Feels like a UI should be added for anytime a feed is unavailable, instead of throwing an error and closing the application.

```
System.InvalidOperationException was unhandled by user code
  HResult=-2146233079
  Message=An error occurred while processing this request.
  Source=Microsoft.Data.Services.Client
  StackTrace:
       at System.Data.Services.Client.BaseAsyncResult.EndExecute[T](Object source, String method, IAsyncResult asyncResult)
       at System.Data.Services.Client.QueryResult.EndExecuteQuery[TElement](Object source, String method, IAsyncResult asyncResult)
       at System.Data.Services.Client.DataServiceRequest.EndExecute[TElement](Object source, DataServiceContext context, String method, IAsyncResult asyncResult)
       at System.Data.Services.Client.DataServiceQuery`1.EndExecute(IAsyncResult asyncResult)
       at ChocolateyGui.Services.PackageServices.ODataPackageService.<>c__DisplayClass3.<GetLatest>b__4(IAsyncResult ar) in C:\ChocolateyGUI\Source\ChocolateyGui\Services\PackageServices\ODataPackageService.cs:line 55
       at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
  InnerException: 
       HResult=-2146233079
       Message=The remote name could not be resolved: 'testserverfeed.com'
       Source=Microsoft.Data.Services.Client
       StackTrace:
            at System.Data.Services.Client.HttpWebRequestMessage.EndGetResponse(IAsyncResult asyncResult)
            at System.Data.Services.Client.ODataRequestMessageWrapper.EndGetResponse(IAsyncResult asyncResult)
            at System.Data.Services.Client.DataServiceContext.GetResponseHelper(ODataRequestMessageWrapper request, IAsyncResult asyncResult, Boolean handleWebException)
            at System.Data.Services.Client.DataServiceContext.EndGetResponse(ODataRequestMessageWrapper request, IAsyncResult asyncResult)
            at System.Data.Services.Client.RequestInfo.EndGetResponse(ODataRequestMessageWrapper request, IAsyncResult asyncResult)
            at System.Data.Services.Client.QueryResult.AsyncEndGetResponse(IAsyncResult asyncResult)
       InnerException: 
            HResult=-2146233079
            Message=The remote name could not be resolved: 'testserverfeed.com'
            Source=System
            StackTrace:
                 at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
                 at System.Data.Services.Client.HttpWebRequestMessage.EndGetResponse(IAsyncResult asyncResult)
            InnerException: 
```

![erroronfeedload2](https://cloud.githubusercontent.com/assets/1831340/5946990/b10cdef6-a6ee-11e4-9e03-9794b56a2e78.png)

**Comment by gep13** [comment]
Thanks for taking the time to report this.  We will look into it.

**Comment by gep13** [comment]
@johnnyramos I have added a basic try/catch around this problem, which should hopefully make it a little clearer about what is going on.  This should be available in the next release 0.13.1.
